content/docs/setup/kubernetes/multicluster/vpn/index.md

```diff
@@ -43,7 +43,7 @@ manifests and Helm charts provided within the Istio repository.
 
 ## Deploy the local control plane
 
-Install the [Istio control plane](/docs/setup/kubernetes/quick-start/#installation-steps)
+Install the [Istio control plane](/docs/setup/kubernetes/install/kubernetes/#installation-steps)
 on **one** Kubernetes cluster.
 
 ## Install the Istio remote
```

Comment:
istio remote install instructions need a rev.  We no longer have an isito-remote chart.

Reply:
Beyond the scope of the PR.